src/requests/get.js

```diff
@@ -50,7 +50,7 @@ async function getAllStackingsAPI(user) {
 }
 
 async function getUserAPI(user) {
-  return GET(`/api/users/${user.uid}`);
+  return GET(`/api/users/${user.uid}`, false);
```

Comment:
All API endpoints now check that the request's user is authenticated, except for this one. I think this is used to see whether they _are_ authenticated, so it doesn't make sense to protect it (though we should double check we're not exposing unnecessary data).